src/components/Display.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Display(prop) {
+  return <h2>{prop.calcResult}</h2>;
+}
+
+export default Display;
```

Comment:
- its calcResult prop should be a String with default value = 0   ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png))   ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))